### PR TITLE
First stab at address specifications

### PR DESCRIPTION
**Pull request by ehoogerbeets**

### CODE REVIEW

File: schemas/address-translations-schema.json [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+                },
+                "Band": {
+                    "type": "string",
+                    "description": "Translation of this field type"
+                },
+                "Borough": {
+                    "type": "string",
+                    "description": "Translation of this field type"
+                },
+                "Burgh": {
```
Comment (gouniLee) [inline schemas/address-translations-schema.json:98]: these property data is pretty specific. where did you get this information?

**Review by gouniLee — APPROVED** [review]

File: schemas/address-translations-schema.json (hunk 1 above)
Comment (ehoogerbeets) [reply schemas/address-translations-schema.json:98]: If you look at the various references at the bottom of specs/Address.md, there is a description of the various types of regions and localities around the world. The "Autonomous Community" and "Band" regions come from the addresses in native American reservations.

Comment (gouniLee) [reply schemas/address-translations-schema.json:98]: oh, thank you.

File: specs/Addresses.md [hunk 2]
```diff
@@ -50,37 +106,78 @@ if they do not explicitly name a parent locale will inherit from the "root" loca
...
+      schema. Incudes a detailed address object for more fine-grained addresses
+  * We also looked at PayPal's i18n reference application and the formats, field labels, and validations offered.
 * Google Address implementations
-    * libaddressinput [AddressValidationMetadata](https://github.com/google/libaddressinput/wiki/AddressValidationMetadata) - describes the address validation metadata Goolge publishes and includes layouts, required fields, postal code validation patterns, field names in English, ISO 3166-2 sub-region codes where appropriate, subregion names, and latin layouts for Asian regions
+    * libaddressinput [AddressValidationMetadata](https://github.com/google/libaddressinput/wiki/AddressValidationMetadata)
+        - describes the address validation metadata Goolge publishes and includes layouts, required fields,
+          postal code validation patterns, field names in English, ISO 3166-2 sub-region codes where
+          appropriate, subregion names, and latin layouts for Asian regions
     * The Google libaddressinput data repository: https://chromium-i18n.appspot.com/ssl-address/data/
```
Comment (gouniLee) [inline specs/Addresses.md]: I think the link might be `https://chromium-i18n.appspot.com/ssl-address`.  If I visit that link it displays `{}` only.

**Pull request merged**